### PR TITLE
Upload only option

### PR DESCRIPTION
**Pull request by sethladd**
First cut. Tried to make the most minimal amount of changes possible. Thanks for taking a look!

### CODE REVIEW

File: lib/src/collect_lcov.dart [hunk 1]
```diff
@@ -85,8 +100,12 @@ class LcovCollector {
     }
   }
 
+  Future<CoverageResult<String>> convertVmReportToLcov({int workers: 1}) async {
```
Comment (kevmoo) [inline lib/src/collect_lcov.dart]: What is this? :smile:

Comment (sethladd) [inline lib/src/collect_lcov.dart]: Good catch! I don't remember :)

**Comment by sethladd** [comment]
Will send another PR

**Comment by sethladd** [comment]
Here's the new PR https://github.com/duse-io/dart-coveralls/pull/42

**Pull request closed without merging**